backend/graphql/submission.graphql

```diff
@@ -147,7 +148,7 @@ input AcceptSubmissionInviteInput @validator(class: "SubmissionInvitationAccepta
     password: String!
 }
 
-input UpdateSubmissionInput {
+input UpdateSubmissionInput @validator(class: "SubmissionUpdateInputValidator") {
```

Comment:
```suggestion
input UpdateSubmissionInput @validator {
```
*if* you rename the validator class to `UpdateSubmissionInputValidator` you won't need to specify the validator class as it'll follow convention.

Reply:
Interesting!

Reply:
Refactored as suggested